.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
```suggestion
A clear, concise description of the problem.
```
Might just be me, but the truncated example didn't seem to provide enough detail to help consumers fill this out. I'd either delete the example or flesh it out.